src/hooks/usePositionsDataTableCompose.js

```diff
@@ -200,6 +200,27 @@ export function usePositionDataTableCompose(positions, confirmActionHandler, ope
    * @returns {JSX.Element} Composed JSX element.
    */
   function renderAge(dataIndex) {
+    const position = positions[dataIndex];
+    const { isCopyTrader, profitSharing, providerOwnerUserId } = position;
+    const currentUserId = storeUserData.userId;
+    const isProviderOwner = providerOwnerUserId === currentUserId;
+
+    if (profitSharing) {
```

Comment:
This could be simplified instead of returning the same result in L210 and 214.
Maybe `profitSharing && !isProviderOwner`

Reply:
Actually currently the age is not displayed in the open positions for PS users. I would get ride of the new renderAge method, use the same as before. 
![image](https://user-images.githubusercontent.com/5314435/118128310-02331f00-b3fb-11eb-8434-612d9f0f7288.png)

Reply:
tried that, but thats not how its wanted. didnt work.